### PR TITLE
Review and update

### PR DESCRIPTION
**Pull request by shavidissa**
Updates the following:
- Formatting of Kubernetes Concepts
- Ensure the page follows the style guide.
- Fix issue https://github.com/kubernetes/website/issues/3735

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -91,32 +91,32 @@ spec:
...
 
-This specification will create a new Service object named “my-service” which
-targets TCP port 9376 on any Pod with the `"app=MyApp"` label.
+This specification creates a new Service object named “my-service”, which
+targets TCP port 9376 on any Pod with the `app=MyApp` label.
 
-This Service will also be assigned an IP address (sometimes called the "cluster IP"),
-which is used by the service proxies
+This Service also assigns an IP address (sometimes called the "cluster IP"),
+which is used by the Service proxies
```
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/service.md]: nit: This specification creates a new Service named "my-service", ...

nit, could the link be cleaned up a bit:
This Service also assigns an IP address, or "cluster IP", which is used by the [Virtual IPs and service proxies](#virtual-ips-and-service-proxies).

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -91,32 +91,32 @@ spec:
...
 Port definitions in Pods have names, and you can reference these names in the
-targetPort attribute of a Service. This will work even if there are a mixture
-of Pods in the Service, with the same network protocol available via different
-port numbers but a single configured name.
+`targetPort` attribute of a Service. This works even if there is a mixture
+of Pods in the Service using a single configured name, with the same network
+protocol available via different port numbers.
 This offers a lot of flexibility for deploying and evolving your Services.
-For example, you can change the port number that pods expose in the next
+For example, you can change the port number the Pods are exposed in the next
```
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/service.md]: nit: Not sure about this change.

File: content/en/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -235,19 +236,19 @@ By default, kube-proxy in userspace mode chooses a backend via a round-robin alg
...
+and redirects that traffic to one of the Service's
 backend sets.  For each Endpoint object, it installs iptables rules which
 select a backend Pod.
 
 By default, kube-proxy in iptables mode chooses a backend at random.
 
 Using iptables to handle traffic has a lower system overhead, because traffic
-is handled by Linux netfilter without the need switch between userspace and the
+is handled by Linux netfilter. It removes the need to switch between userspace and the
 kernel space. This approach is also likely to be more reliable.
```
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/service.md]: What is the `it`, iptables or Linux `netfilter`?

File: content/en/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -291,22 +292,22 @@ these are:
 To run kube-proxy in IPVS mode, you must make the IPVS Linux available on
 the node before you starting kube-proxy.
 
-When kube-proxy starts in IPVS proxy mode, it will verify whether IPVS
-kernel modules are available, and if those are not detected then kube-proxy
+When kube-proxy starts in IPVS proxy mode, it verifies whether IPVS
+kernel modules are available, and if those are not detected, then kube-proxy
```
Comment (kbhawkey) [inline content/en/docs/concepts/services-networking/service.md]: nit: fall vs. falls
nit: could the sentence be broken up?

File: content/en/docs/concepts/services-networking/service.md (hunk 2 above)
Comment (shavidissa) [reply content/en/docs/concepts/services-networking/service.md]: This was fixed by a previous commit. Thank you!

File: content/en/docs/concepts/services-networking/service.md (hunk 3 above)
Comment (shavidissa) [reply content/en/docs/concepts/services-networking/service.md]: true.. That's confusing.. am reverting back to the old sentence and did a small nit there.

File: content/en/docs/concepts/services-networking/service.md (hunk 2 above)
Comment (zacharysarah) [reply content/en/docs/concepts/services-networking/service.md]: Yup, 🚫 passive voice.

**Comment by shavidissa** [comment]
Hi @kbhawkey and team,

Will you be able to help lgtm and approve the PR if the content is good to go, please? :)

**Comment by kbhawkey** [comment]
Preview page:
https://deploy-preview-15074--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/service/

**Comment by kbhawkey** [comment]
hello @shavidissa. I read through the changes. Seems fine. Nice cleanup.
I think this concept page, though, could use some updates in a future PR, such as:
- reduce the size of the diagrams and fix the overflow of the text in the diagrams
- create a separate page or address the AWS specific content?
- reword the `Motivation` section -- the description of `The Life of a Pod` gives me a chuckle. A paragraph about `What is a Service` seems more appropriate.
- Are there more links that could be added to `Future work`?

**Comment by zacharysarah** [comment]
@kbhawkey 👋 I'm giving the LGTM/approve here to keep things moving, but your feedback here is great and I don't want it to get lost.

**Pull request merged**